thunder/__init__.py

```diff
@@ -211,6 +211,19 @@ def _get_cache_info():
     return _cache_info_ctx.get()
 
 
+def _get_thunder_module(model):
```

Comment:
Aren't you duplicating this? Since you moved it to a different module in the previous PR